src/Fantomas.Core.Tests/ColMultilineItemTests.fs

```diff
@@ -135,8 +135,8 @@ let b (x: int) (y: int) : int =
 let c () =
     try
         0
-    with
-    | ex -> 1
+    with ex ->
```

Comment:
Should the body of the `SynMatchClause` always be on the next line, indented?

Reply:
Yes